$OpenBSD: patch-plugins_vidinput_bsd_vidinput_bsd_cxx,v 1.3 2021/11/28 13:31:14 ajacoutot Exp $

vidinput_bsd.cxx:311:21: error: ordered comparison between pointer and zero

Index: plugins/vidinput_bsd/vidinput_bsd.cxx
--- plugins/vidinput_bsd/vidinput_bsd.cxx.orig
+++ plugins/vidinput_bsd/vidinput_bsd.cxx
@@ -308,7 +308,7 @@ PBoolean PVideoInputDevice_BSDCAPTURE::GetFrameDataNoD
 
     mmap_size = frameBytes;
     videoBuffer = (BYTE *)::mmap(0, mmap_size, PROT_READ, 0, videoFd, 0);
-    if (videoBuffer < 0) {
+    if (videoBuffer == MAP_FAILED) {
       return PFalse;
     } else {
       canMap = 1;
